### DISCUSSION TITLE
GitHub Student Benefits

### DISCUSSION THREAD

**Discussion by Ozanelon** [community/community#135405] [GitHub Education] [answered]
### Hi everyone,

I approved for the GitHub Education Program 3 days ago. But I still haven't received my student benefits.

**Comment by davevad93** [comment, marked as answer]
Hi @Ozanelon ,

The waiting time may vary depending on different factors, such as the volume of requests. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks. 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Ozanelon was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135145

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by Chinmai2001** [reply]
I got my benefits today, But i am not able to access the third party benefits like data camp. Help me with this..
Thank you.

**Reply by davevad93** [reply]
Hey @Chinmai2001, I have replied to your discussion with a solution here #154960.

**Comment by jammylaw** [comment]
I've tried to apply for the student pack but keep getting rejected. Although I had a change of name which my conflicts with of of my information but I have since had all the necessary documentation of my new name like School id card, school email, admissions letter. But still getting rejected. Pls help.
